scripts/build.py

```diff
@@ -112,7 +112,7 @@
     print("subpath: " + development_subpath)
     print("type: " + build_type)
     pkgutil.collect_plugin("hdr-switch", os.path.join(switch_rom_path, development_subpath), build_type, "development.nro", "development")
-    pkgutil.collect_plugin("hdr-ryujinx", os.path.join(ryujinx_rom_path, development_subpath), build_type, "development.nro", "development")
+    pkgutil.collect_plugin("hdr-ryujinx", os.path.join("sdcard", os.path.join(switch_rom_path, development_subpath)), build_type, "development.nro", "development")
```

Comment:
this can just be `os.path.join("sdcard", switch_rom_path, development_subpath)` without the nested join